src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEPropertySymbol.cs

```diff
@@ -784,6 +785,7 @@ internal override UseSiteInfo<AssemblySymbol> GetUseSiteInfo()
             {
                 var result = new UseSiteInfo<AssemblySymbol>(primaryDependency);
                 CalculateUseSiteDiagnostic(ref result);
+                DeriveUseSiteInfoFromCompilerFeatureRequiredAttributes(ref result, Handle, allowedFeatures: CompilerFeatureRequiredFeatures.None);
```

Comment:
>DeriveUseSiteInfoFromCompilerFeatureRequiredAttributes(ref result, Handle, allowedFeatures: CompilerFeatureRequiredFeatures.None); [](http://example.com/codeflow?start=16&length=130)

Are we intentionally not inferring anything from parameters?

Reply:
> Are we intentionally not inferring anything from parameters?

I would expect parameters to be reported by the method symbol for the getter/setter.